icpc-wf/ansible/roles/cds/handlers/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Define here handlers associated to this role.
+
+- name: restart systemctl
+  shell: systemctl daemon-reload
```

Comment:
`command` is slightly saver I think.

Reply:
This is a copy paste from other places. Let's change it everywhere at once (in a separate PR)